<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NGA萌战计票工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>选手信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>昵称</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QPushButton" name="addGunPushButton">
           <property name="text">
            <string>增加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="delGunPushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="gunLineEdit"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QPushButton" name="addNamePushButton">
           <property name="text">
            <string>增加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="delNamePushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="gunListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="nameListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>统计信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="voteTableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>帖子TID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tidLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>页数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pagesLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>赛程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scheduleLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>真爱倍率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="trueloveLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="startPushButton">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadSettingPushButton">
        <property name="text">
         <string>导入配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveSettingPushButton">
        <property name="text">
         <string>导出配置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
